tensorflow/tsl/platform/cpu_info.cc

```diff
@@ -345,6 +352,115 @@ void InitCPUIDInfo() {
 
 #endif  // PLATFORM_IS_X86
 
+#ifdef PLATFORM_IS_ARM64
+
+class CPUIDInfo;
+void InitCPUIDInfo();
+
+CPUIDInfo *cpuid = nullptr;
+
+// Structure for basic CPUID info
+class CPUIDInfo {
+ public:
+  CPUIDInfo()
+      : implementer_(0),
+        variant_(0),
+        cpunum_(0),
+        is_arm_neoverse_v1_(0),
+        is_arm_neoverse_n1_(0) {}
+
+  static void Initialize() {
+    // Initialize cpuid struct.
```

Comment:
Nit:
```suggestion
    // Initialize CPUIDInfo pointer.
```

Reply:
Fixed in https://github.com/tensorflow/tensorflow/pull/61110/files